network-interface/src/request/mod.rs

```diff
@@ -220,24 +220,20 @@ pub fn request_handler<T: Send + Sync + Clone + 'static, Req: Handle<N, T>, N: N
                     let req_environment = req_environment.clone();
                     let network = Arc::clone(&network);
 
-                    tokio::spawn(async move {
```

Comment:
There's the same kind of spawn in the `message_handler` function below. Maybe that can be removed, too?

Reply:
That one looks unnecessary  too